src/applications/static-pages/covid-vaccine-updates-cta/createCovidVaccineUpdatesWidget.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+export default function createCovidVaccineUpdatesWidget(store, widgetType) {
+  const root = document.querySelector(`[data-widget-type="${widgetType}"]`);
+  if (root) {
+    import(/* webpackChunkName: "CaregiverContentToggle" */
```

Comment:
Confirming here that the webpack chunk name is correct

Reply:
just pushed a rename!